### ISSUE TITLE
[BUG] The 'well-known/smart-configuration' response does not include 'capabilities'

### ISSUE THREAD

**Issue by Soyoung7533** [Aidbox/Issues#433]
**Describe the bug**
The [hl7FHIR](https://hl7.org/fhir/smart-app-launch/conformance/index.html#metadata) states that the 'capabilities' property is required in the response, however, the AidBox response body does not include it.

**Severity**
S0 - Many tests in the Touchstone Conformance Patient API test suites (FHIR4-0-1-PAA-SMART-PAT-PUBLIC-Server, FHIR4-0-1-PAA-SMART-PAT-CONFID-Server, etc..) fail due to this issue or are dependant on it

**Steps to reproduce the behavior:**
Request: GET /well-known/smart-configuration

**Expected behavior**
The 'well-known/smart-configuration' response body includes 'capabilities'

**Screenshots**
<img width="1169" alt="smart-configuration response" src="https://user-images.githubusercontent.com/89491379/132604803-e6792220-a7d3-4fb1-86d0-24b49668d7f6.png">

**Versions:**
- OS version

- Docker version (if applicable)

 - Aidbox image version
 

 - Aidboxdb version
    Stable

 - Browser version (if applicable)

 - Cloud provider (if applicable)

**Additional context**
Touchstone Conformance Test

**Comment by Nesmeshnoy** [comment]
Please use GET smart/.well-known/smart-configuration to run tests

**Comment by Soyoung7533** [comment]
As I mentioned in the support chat, this issue doesn't seem to have been fixed.

`Hi, I installed devbox:edge v:2201.3b132cfb, and set up the smart on fhir as per https://github.com/Aidbox/aidbox-project-samples#smart-on-fhir-aidbox-installation. The response of the /.well-known/smart-configuration request includes an empty set of "capabilities”. Can I configure the capabilities in the smart-configuration response?`

[smart-configuration response.json.txt](https://github.com/Aidbox/Issues/files/7974454/smart-configuration.response.json.txt)

Both _/smart/.well-known/smart-configuration_ and _/.well-known/smart-configuration_ does not include the capabilities.
